pkg/clusterconditions/promql/promql.go

```diff
@@ -50,9 +50,10 @@ func NewPromQL(kubeClient kubernetes.Interface) *cache.Cache {
 			},
 			QueryTimeout: 5 * time.Minute,
 		},
-		MinBetweenMatches: 10 * time.Minute,
-		MinForCondition:   time.Hour,
-		Expiration:        24 * time.Hour,
+		MinBetweenMatchesInitial:    1 * time.Second,
+		MinBetweenMatchesWhenCached: 10 * time.Minute,
```

Comment:
Why not keeping the value of `MinBetweenMatchesWhenCached` at a low value e.g. 1 minute and not 10 mins?

Reply:
discussed internally, and as summarized in 50bf745af1b68005a9056d442289930b5a67d47a , we've decided to drop the `MinBetweenMatchesInitial` / `MinBetweenMatchesWhenCached` approach and just move to `MinBetweenMatches=1s`.